internal/client/oidc/oidc.go

```diff
@@ -256,6 +264,45 @@ func (o *OidcClient) refresh(issuer string, clientID string) error {
 // It presents a user code for the end user to input in the device that has web access and waits for them to complete the authentication,
 // subsequently updating the client's tokens upon successful authentication.
 func (o *OidcClient) authenticate(issuer string, clientID string, audience string) error {
```

Comment:
Does this ever trigger from the CLI anymore? Since we now send all requests to the daemon only, I wonder if this should be removed?

Reply:
Yes, when we have OIDC authentication configured for the `migration-manager` client itself back to the `migration-managerd` daemon.